doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* Four eye principle on the organisation level.
+
+  * Changes or additions to common and shared code shall have reviews from
```

Comment:
I think it wouldn't hurt defining "common and shared code". Is that strictly equivalent to non vendor specific code ?

Reply:
sure